modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -47,8 +54,17 @@ endif::restricted[]
 Complete the following steps on the installation host:
 
 ifndef::openshift-origin[]
+
+// TODO: Is this the part that needs to change for oc-mirror? That this has to be saved to ~/.docker/config.json ?
+ifndef::oc-mirror[]
 . Download your `registry.redhat.io` pull secret from the
 link:https://console.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {console-redhat-com} site and save it to a `.json` file.
+endif::[]
+ifdef::oc-mirror[]
+. Download your `registry.redhat.io` pull secret from the
+link:https://console.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {console-redhat-com} site and save it to `~/.docker/config.json`.
```

Comment:
@afflom @jpower432 Can you confirm whether these changes are what we needed to note as the difference in the credentials procedure for oc-mirror? Instead of saving the pull secret in step 1 to an arbitrary .json file, we want to have them save it to `~/.docker/config.json`? 

Though honestly the steps as they are now don't look quite right to me. Step 1 downloads it but as a text file, and then step 3 references some text file that wasn't actually created anywhere in the procedure. Should the real steps be to:

1. Download the pull secret as a text file.
2. Transform that pull secret text file into JSON format. (And for oc-mirror, that should be saved to `~/.docker/config.json`?
3. Then, move forward with getting the username/password (originally step 2)
4. Follow the rest of the steps

Appreciate any thoughts you have on this.

Preview of the section as it is now in the live docs: https://deploy-preview-40474--osdocs.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-installation-images.html#installation-adding-registry-pull-secret_installing-mirroring-installation-images

Preview of the section for the oc-mirror docs, w/ my updates here: https://deploy-preview-40474--osdocs.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#installation-adding-registry-pull-secret_installing-mirroring-disconnected

Reply:
I agree with you there, the steps sound a bit confusing. I agree that steps 2 and 3 should be reversed. We actually also support the default `podman` credential location of `$XDG_RUNTIME_DIR/containers/auth.json` as well